src/main/webapp/mapStyle.css

```diff
@@ -229,3 +229,15 @@ body, html {
 #shareBtn {
   background-image: url("icons/share.svg");
 }
+
+#search {
+  left: 280px !important;
```

Comment:
nit: Instead of using `!important`, it's better to increase the specificity of your selector. For example, you can use `#mapOverlay #search` instead of `#search`. That selector has an ID specificity of 2, so it will override other rules that have a specificity of 1.

Reply:
I just tried using `#mapoverlay #search` but it didn't work. It seems that the API is adding style properties directly to the `style` attribute of the DOM element which overrides any CSS styles. When I remove the style attribute in the console, the CSS styling works with only `#search` so I'll try adding a `domready` listener to the DOM element and then remove the API styling then.

Reply:
I can't seem to find a solution at the moment so I'll leave it as is and come back to it.